### DISCUSSION TITLE
Translation mistake

### DISCUSSION THREAD

**Discussion by mefjak** [Moonif/MacBox#10] [Q&A] [answered]
If I see an error with translation - should I just open an issue or how can i report it? On discussion?

**Comment by Moonif** [comment, marked as answer]
Hi,

You can do it here if you want.

Thanks

**Reply by mefjak** [reply]
Cool.

So there is a mistake in polish translation.

There is a string "Nazwisko" which should be "Nazwa"
Nazwa = Name
Nazwisko = Surname

![Zrzut ekranu 2024-02-24 o 17 41 15](https://github.com/Moonif/MacBox/assets/22888465/aa6bc2d7-aa67-457a-bcc0-531363cd5c28)

**Reply by Moonif** [reply]
Okay! I'll change it, thank you. 👍

**Reply by mefjak** [reply]
No problem :)
